scripts/makeNetwork.py

```diff
@@ -147,7 +148,8 @@ def qemuArchNetworkConfig(i, arch, n):
         else:
             vlan_id = vlan if vlan else i
             mac_str = "" if not mac else ",macaddr=%s" % mac
```

Comment:
The `",macaddr=%s"` part should be changed to ",mac=%s'

Reply:
Fixed  in the commit `bf57905`.